Flight/PassengerOperations.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AirportTicketBookingSystem.Enums;
+using AirportTicketBookingSystem.UserRole;
+
+namespace AirportTicketBookingSystem.Flights
+{
+    public class PassengerOperations
+    {
+        public FlightManagement flightManagement = new FlightManagement();
+
+        public void SelectPassengerOperation(Passenger passenger)
+        {
+            Console.Clear();
+            Console.WriteLine($"HELLO {passenger.Name}, Welcome to Booking System!");
+
+            while (true)
+            {
+                PasengerMenu();
+                Console.Write("Enter your choice: ");
+                PassengerOperation choice;
+                while (!Enum.TryParse(Console.ReadLine(), out choice))
+                {
+                    Console.WriteLine("Invalid input. Please enter a valid operation.");
+                    Console.Write("Enter your choice: ");
+                }
+
+                switch (choice)
+                {
+                    case PassengerOperation.BookFlight:
+                        flightManagement.BookFlight();
+                        break;
+
+                    case PassengerOperation.ListAllFlights:
+                        flightManagement.ListAllFlights();
+                        break;
+
+                    case PassengerOperation.CancelBooking:
+                        flightManagement.CancelBooking();
+                        break;
+
+                    case PassengerOperation.EditBooking:
+                        flightManagement.EditBooking();
+                        break;
+
+                    case PassengerOperation.ViewPersonalBookings:
+                        flightManagement.ViewPersonalBookings();
+                        break;
+
+                    case PassengerOperation.SearchBooking:
```

Comment:
here you used the filtering method but it shouldn't, there is a big difference between Search & Filter in the Business layer, 
Search: I have a complex criterion with different properties that should be all filled and all matched on the criteria properties, all property should be filled.
Filter: I have single criteria property so I will pass one property to get all matched, one parameter is enough